### PR TITLE
Update handler files

### PR DESCRIPTION
**Pull request by pmuens**
## What did you implement:

***Implementing Issue:*** #2256

This PR adds the new handlers so that they work well with the recent API Gateway PROXY implementation (see: #2185).

## How did you implement it:

Just added the new handler files.

## How can we verify it:

`serverless create -t aws-nodejs -p my-new-service`
`serverless deploy`
`curl <url>`

## Todos:

- [x] Update handler file for Node.js
- [x] Update handler file for Python
- [x] Write documentation
- [x] Fix linting errors
- [x] Make sure code coverage hasn't dropped
- [x] Provide verification config/commands/resources
- [x] Change ready for review message below

***Is this ready for review?:*** YES

### CODE REVIEW

**Comment by pmuens** [comment]
@flomotlik just updated the handler files for Node.js and Python.

Was tinkering around with Java but was not successful by now (maybe you can look into it if you have some time). Otherwise I'll just reverse engineer it more.

File: lib/plugins/create/templates/aws-nodejs/handler.js [hunk 1]
```diff
@@ -1,8 +1,24 @@
...
+    message: 'Go Serverless v1.0! Your function executed successfully!',
+    input: event,
+  };
+
+  const response = {
+    statusCode: 200,
+    headers: {
+      'custom-header': 'Custom header value',
+    },
+    body: JSON.stringify(body),
```
Comment (eahefnawy) [inline lib/plugins/create/templates/aws-nodejs/handler.js]: why do we `JSON.stringify` here? isn't that handled by AWS? if not, then shouldn't we stringify the entire `response` object?

File: lib/plugins/create/templates/aws-python/handler.py [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+        "message": "Go Serverless v1.0! Your function executed successfully!",
+        "input": event
+    }
+
+    response = {
+        "statusCode": 200,
+        "headers": {
+            "custom-header": "Custom header value"
+        },
+        "body": json.dumps(body)
```
Comment (eahefnawy) [inline lib/plugins/create/templates/aws-python/handler.py]: same goes here

**Comment by flomotlik** [comment]
@pmuens I'll take care of Java and Scala today

**Comment by flomotlik** [comment]
@pmuens can you also look through the onboarding guide to make sure its up to date and has a mention why we're adding this code.

**Comment by pmuens** [comment]
@flomotlik @eahefnawy just updated the PR according to your feedback.

Please review and let me know if there's anything else which should be done here...

**Pull request merged**